types/schema/Metadata.ts

```diff
@@ -24,6 +25,16 @@ export interface BaseMetadata {
   schema: URL;
 }
 
+/**
+ * @id #ServiceFiles
+ * @description File types associated with this service. Schema["files"] will be a subset of this list based on the user's journey through the service.
+ */
+export interface ServiceFiles {
```

Comment:
nit: I really liked the name `RequestedFiles` from one of your earlier proposals - would that perhaps be more descriptive here as "Service" is already implied based on position in schema? Then `@description` "File types requested by this service" rather than "associated with" ?

Reply:
Yep totally agree! I've actually changed this back locally also - `RequestedFiles` is significantly clearer.

Reply:
Changes force-pushed 👍